### ISSUE TITLE
//DEPS content assist only works on first item

### ISSUE THREAD

**Issue by quintesse** [jbangdev/jbang-idea#54]
When you have a line like

```
//DEPS io.javalin:javalin:4.4.0 org.slf4j:slf4j-simple:1.7.36
```

And you put your cursor after org.slf4j:slf4j-simple:" and hit Ctrl+Space it shows suggestions for "io.javalin:javalin:", not "org.slf4j:slf4j-simple:".

**Comment by linux-china** [comment]
I think one `//DEPS ` just for one dependency.  I will fix it. 

```
//DEPS io.javalin:javalin:4.4.0 
//DEPS org.slf4j:slf4j-simple:1.7.36

```

**Comment by quintesse** [comment]
Yeah, you can have more than one dependency on a single line and they can be either comma or space separated.

**Comment by maxandersen** [comment]
This should be addressed by the rewrite in PR #159 (version 0.100.0). Please try it out and reopen if the issue persists — feedback welcome!
